src/patternfly/components/DualListSelector/examples/DualListSelector.md

```diff
@@ -676,6 +855,11 @@ cssPrefix: pf-c-dual-list-selector
 | `.pf-c-dual-list-selector__item-text` | `<span>` | Initiates a dual list selector pane menu item text. **Required** |
 | `.pf-c-dual-list-selector__controls` | `<div>` | Initiates the dual list selector controls. **Required** |
 | `.pf-c-dual-list-selector__controls-item` | `<div>` | Initiates the dual list selector controls item. **Required** |
+| `.pf-c-dual-list-selector__item-check` | `<span>` | Initiates the dual list selector item check. |
```

Comment:
can you move these `__item-` elements above `__controls` with the other `__item-` things?